.devops/repo/src/commands/bump.rs

```diff
@@ -0,0 +1,70 @@
+use crate::utils::print_applications;
+use std::process::Command;
+
+pub(crate) fn run(apps: Vec<String>, root: String) {
+    let mut success: Vec<String> = vec![];
+
+    let toolchain = "fuel-toolchain.toml".to_string();
+    let tmp_toolchain = "fuel-toolchain2.toml".to_string();
+
+    for app in apps {
+        println!("\nBumping {}", app);
+
+        // TODO: safety
+        let project = std::fs::canonicalize(format!("{}/{}/project", root, app)).unwrap();
+
+        let _ = Command::new("mv")
+            .env("IFS", "''")
+            .args([
+                format!("{}/{}", project.clone().display(), toolchain.clone()),
+                format!("{}/{}", project.clone().display(), tmp_toolchain.clone()),
+            ])
+            .status();
+
+        let _ = Command::new("cp")
+            .env("IFS", "''")
+            .args([
+                format!("./{}", toolchain.clone()),
+                format!("{}/{}", project.clone().display(), toolchain.clone()),
+            ])
+            .status();
+
+        let result = Command::new("forc")
+            .current_dir(project.clone())
+            .arg("build")
+            .status();
+
+        match result {
```

Comment:
This can also be

```suggestion
        match Command::new("forc")
            .current_dir(project.clone())
            .arg("build")
            .status() {
```